bin/start_fe.sh

```diff
@@ -176,6 +176,11 @@ if [ ! -d $LOG_DIR ]; then
     mkdir -p $LOG_DIR
 fi
 
+meta_dir=$STARROCKS_HOME/meta
```

Comment:
This is not good enough, metadir can be changed in `fe.conf` by `meta_dir = <somewhere else>`. and also it could be possible that the user who lanches the process, has no permission to create the meta directory.

Reply:
I added `META_DIR` uppercase variable in `fe.conf`,  it is similar to `LOG_DIR`. Both `META_DIR` and `LOG_DIR` have not considered the user permission problem.  But I am not sure whether it is ok to introduce a new environment variable in `fe.conf` and `start_fe.sh`.